hack/jenkins/e2e.sh

```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Namespaces.*should\sdelete\sfast"
     "PD"
     "ServiceAccounts"
-    "Services.*change\sthe\stype"
```

Comment:
this one flaked this morning: https://goto.google.com/k8s-test/view/Flaky/job/kubernetes-e2e-gce-parallel-flaky/7453/